client/src/pages/Configuration/Rules/Rules.tsx

```diff
@@ -0,0 +1,324 @@
+                Organizator zastrzega sobie prawo do nieuwzględnienia wyników danego Podmiotu i nieprzyznania Upominków za dany okres
+                Uczestnikom.
+              </Text>
+              <Text>
+                13. W przypadku nieodebrania upominków przez przedstawiciela danego Podmiotu za wskazany okres w terminie 10 dni roboczych
+                od otrzymania od Organizatora informacji o takiej możliwości, Organizator zastrzega sobie prawo do niewydania upominków w
+                terminie późniejszym. W takiej sytuacji upominki wrócą do ogólnej puli.
+              </Text>
+              <Text>
+                14. W przypadku powtarzających się opóźnień w odbiorze upominków ze strony Podmiotu Organizator może wykluczyć dany Podmiot
+                z dalszego udziału w Kampanii.
+              </Text>
+              <Title order={4}>§ 12 Postanowienia końcowe</Title>{' '}
+              <Text>
+                1. Organizator nie ponosi odpowiedzialności za: - skutki prawne działań podjętych przez Podmiot na rzecz przeprowadzenia
+                Kampanii wśród swoich pracowników, w tym za ewentualne konsekwencje prawne wynikające z korzystania z aplikacji lub innych
+                narzędzi elektronicznych na potrzeby rejestracji przejazdów, - następstwa działań lub zdarzeń będących konsekwencjami
+                uczestnictwa w Kampanii (np. odpowiedzialność odszkodowawcza, konsekwencje zdrowotne, prawne, finansowe, szkody na mieniu i
+                ciele, zniszczenia, kradzieże itp.), - zaginięcie/zniszczenie upominków w przypadku przekazania upominków Koordynatorowi
+                wewnętrznemu w sposób inny niż bezpośredni (niebezpośredni sposób przekazania upominków może być uzgodniony między
+                Organizatorem a Koordynatorem wewnętrznym na prośbę Koordynatora wewnętrznego), - przebieg Kampanii w ramach danego Podmiotu
+                (np. za jej promocję), w tym za pracę Koordynatorów wewnętrznych.
+              </Text>
+              <Text>
+                2. Organizator zaleca udział w Kampanii z uwzględnieniem możliwości zdrowotnych i fizycznych każdego z Uczestników.
+              </Text>
+              <Text>
+                3. Organizator dopuszcza zmianę niniejszego Regulaminu, jeśli zaistnieją do tego stosowne przesłanki. O takiej zmianie
+                Organizator poinformuje Koordynatorów wewnętrznych w najkrótszym możliwym terminie.
+              </Text>
+              <Text>
+                4. Prawo do podjęcia decyzji w sprawach nieuregulowanych niniejszym Regulaminem oraz prawo do interpretacji zapisów
+                Regulaminu posiada Organizator.
+              </Text>
+            </Text>
+          </TypographyStylesProvider>
+        </ScrollArea>
+      </Modal>
+      <Button sx={{ padding: 0 }} variant="subtle" onClick={() => setOpened(true)}>
+        Akceptuję regulamin konkursu
+      </Button>
```

Comment:
Ten przycisk to ja bym usunął, bo on i tak checkboxa nie zaznacza. Ewentualnie można poprawić tak by zaznaczał :)